.github/workflows/github-labels.yml

```
@@ -0,0 +1,169 @@
+name: Apply Standardized GitHub Labels
+
+on:
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)